docs/README.md

```diff
@@ -280,24 +280,34 @@ aws cloudfront  create-invalidation --profile docs.docker.com --distribution-id
 aws cloudfront  create-invalidation --profile docs.docker.com --distribution-id $DISTRIBUTION_ID --invalidation-batch '{"Paths":{"Quantity":1, "Items":["/v1.1/reference/api/docker_io_oauth_api/"]},"CallerReference":"6Mar2015sventest1"}'
 ```
 
-### Generate the man pages for Mac OSX
+### Generate the man pages 
 
-When using Docker on Mac OSX the man pages will be missing by default. You can manually generate them by following these steps:
+On Linux installations, Docker includes a set of man pages you can access by typing `man command-name` on the command line. For example, `man docker` displays the `docker` man page. When using Docker on Mac OSX the man pages are not automatically included. 
 
-1. Checkout the docker source. You must clone into your `/Users` directory because Boot2Docker can only share this path
-   with the docker containers.
+You can generate and install the `man` pages yourself by following these steps:
+
+1. Checkout the `docker` source. 
 
         $ git clone https://github.com/docker/docker.git
+        
+  If you are using Boot2Docker, you must clone into your `/Users` directory
+  because Boot2Docker can only share this path with the docker containers.
 		
 2. Build the docker image.
    
-        $ cd docker/docs/man
+        $ cd docker/man
         $ docker build -t docker/md2man .
 
 3. Build the man pages.
 
-        $ docker run -v /Users/<path-to-git-dir>/docker/docs/man:/docs:rw -w /docs -i docker/md2man /docs/md2man-all.sh
+        $ docker run -v /Users/<path-to-git-dir>/docker/man:/man:rw -w /man -i docker/md2man /man/md2man-all.sh
```

Comment:
I'd prefer to assume people could figure that out themselves. Either that or replace `/Users/<path-to-dir>/docker/man:/man` with just `$PWD:/man`  since they should be in the 'man' dir already based on the above steps.